plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -170,6 +170,31 @@ public void testCreateAndDropTableWithSpecialCharacterName(String tableName)
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    // TODO remove this test after refactoring testCreateAndDropTableWithSpecialCharacterName
+    @Override
+    @Test(dataProvider = "testTableNameDataProvider")
```

Comment:
This doesn't ensure we have _better_ test coverage, since the `super.testRenameColumnName` data provider can change and have _more_ test cases.

What about adding separate test method `testRenameColumnNameAdditionalTests` for now?
this would mean we test twice, but it's a fair price for the fact we didn't add additional test coverage in "the right place" in https://github.com/trinodb/trino/pull/14286 and would emphasize the need for cleaning this up

Reply:
done